### ISSUE TITLE
ReactOS doesn't boot

### ISSUE THREAD

**Issue by copy** [copy/v86#52]
See also: https://jira.reactos.org/browse/CORE-8803

Seems to be mostly because of imprecise IDE emulation, but feature-wise, it should work.

**Comment by copy** [comment]
` Assertion 'GuardedMutex->Owner != Thread' failed` is gone now, but it still fails with disk operations:

```
(drivers/filesystems/cdfs/fsctl.c:469) CdfsVerifyVolume() called
(drivers/filesystems/cdfs/fsctl.c:488) Device object B0231E20  Device to verify B02355A0
(drivers/filesystems/cdfs/fsctl.c:497) Same volume!
(drivers/filesystems/cdfs/common.c:120) Failed to read after successful verify, aborting
```

**Comment by warpcoil** [comment]
I have got partially Microsoft&reg; Windows&trade; apps working though!

**Comment by Creeper20428** [comment]
@warpcoil On where?

**Comment by Creeper20428** [comment]
I think this might help.
JDosBox seems to be able to boot ReactOS and some form of Linux (I only tried the ReactOS demo)
jdosbox.sourceforge.net/
It seems pretty fast, probably because of the dynarec core.
It might be interesting to see what kind of code they use. I took a peek and it seems really well-written.
Unfortunately, it doesn't seem to be developed any more (last commit was May 2014, almost 2 years ago!)
